app/controllers/lists_controller.rb

```diff
@@ -1,6 +1,12 @@
 class ListsController < ApplicationController
+  before_filter :authenticate_user!
+  before_action only: [:index, :new, :edit, :show, :create, :update, :destroy_multiple] do
```

Comment:
if it applies to every action, you can just omit the `only` config setting entirely and just have `before_action do`